scripts/helpers/wallet-precalculation.helper.js

```diff
@@ -125,16 +125,11 @@ export class WalletPrecalculationHelper {
         networkName: config.network,
         accountDerivationIndex,
       });
-      const addresses = walletUtils.getAddresses(
```

Comment:
We have `loadAddresses` to derive a range of addresses from the storage instance, it determines if the addresses should be P2PKH or Multisig and we also have methods to generate addresses be their index (e.g. `deriveAddressFromXPubP2PKH` and `deriveAddressFromDataP2SH`).

Since the wallet precalculation helper only generates P2PKH wallets it makes sense to have a method to generate a range of P2PKH addresses but what do you think of having the same loop here and use `deriveAddressFromXPubP2PKH` instead?